Packs/Active_Directory_Query/ReleaseNotes/1_6_18.md

```diff
@@ -0,0 +1,6 @@
+
+#### Integrations
+
+##### Active Directory Query v2
+- %%UPDATE_RN%%
```

Comment:
Please add here an explanation regarding the new command. For more information please see: https://xsoar.pan.dev/docs/concepts/demisto-sdk#update-release-notes.

Reply:
Updated in this commit: 62eda0c435d1a8d202776b6f9711caa2aa6c63e3